doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -594,6 +594,159 @@ must be sent.
 *default location*, then the query will be declared as ambigous and an HTTP error response with code ``409``
 must be sent. 
 
+## Notifications formats
+
+Notifications include two fields: `subscriptionId` (which value is the one of the subscription triggering
```

Comment:
it could be read better if it is described using bullet points:

Notifications include two fields:

* 'subscriptionId' which represents the concerned subscription that originates the notification
* 'data' the notification data itself which includes the entity and attributes concerned